src/lgfx/v1/Touch.hpp

```diff
@@ -67,14 +67,15 @@ namespace lgfx
         };
       };
       int16_t pin_cs = -1;
+      bool is_SPI = false;
```

Comment:
I don't think this change at the interface is needed. The interface has a way to determine if it's spi or i2c by querying the value of cs. 
Rather than changing this interface the application that uses this interface has to be changed.

Reply:
That might work. 0 is a valid pin number, but not all of them set to it. There is the problem of the union, which is in itself a really weird thing. So if we set the SPI elements to 0, it overwrites the I2C elements.